index.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" type ="text/css" href="style.css" />
+    <title>Youtube Microverse</title>
+  </head>
+  <body>
+  <nav class="topnav">
```

Comment:
You have over declared Nav tag. To keep it simple, just think of the **nav** as wraps for your navigations. In this context, you only need one nav tag to handle the elements in your navigation bar.😔
you have the same errors repeated all over the code.